src/util/lang.js

```diff
@@ -63,6 +63,17 @@ exports.delete = function (obj, key) {
 }
 
 /**
+ * Check whether the object has the property.
+ *
+ * @param {Object} obj
+ * @param {String} key
+ * @return {Boolean}
+ */
+exports.hasOwnProperty = function (obj, key) {
+  return Object.prototype.hasOwnProperty.call(obj, key)
```

Comment:
No, what I mean is just hold it in a variable outside the function as `var hasOwn = Object.prototype.hasOwnProperty`
It probably doesn't make much difference, but saves two property lookups on every call ;)